js/service/shell/index.js

```diff
@@ -45,3 +46,19 @@ exports.setCommand = function(name, cb) {
   assert(typeutils.isFunction(cb));
   commands.set(name, cb);
 };
+
+exports.runCommand = function(name, args) {
```

Comment:
How can command stdio be redirected? We could pass all input data as env for this command to use:
```js
env = {
  stdin,
  stdout,
  args
}

function(name, env, cb) {
}
```

Do we need a callback here to know when it's finished?